src/TextInput/TextInput.svelte

```diff
@@ -94,6 +94,7 @@
     dispatch("change", parse(e.target.value));
   };
 
+  // svelte-ignore reactive_declaration_non_reactive_property
   $: isFluid = !!ctx && ctx.isFluid;
```

Comment:
Same as the password component.
```suggestion
  const isFluid = !!ctx && ctx.isFluid;
```